[2024-04-05T16:09:17.456+0000] {taskinstance.py:1159} INFO - Dependencies not met for <TaskInstance: parallel_dag.extract_a scheduled__2024-04-03T00:00:00+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-04-05T16:09:17.663+0000] {local_task_job.py:103} INFO - Task is not able to be run
